src/generic_ui/scripts/content_digitalocean.js

```diff
@@ -1,4 +1,49 @@
-(function improveDOWelcomeFlow(document, navigator) {
+/*                       _____________________________
```

Comment:
@trevj Here's the commit where I fixed the show-overlay logic. DigitalOcean apparently changed their /welcome flow -- now they redirect straight to /droplets upon adding a payment method, rather than back to the /welcome page.